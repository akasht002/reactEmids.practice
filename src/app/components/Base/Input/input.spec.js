import React from 'react';
import {Input} from './index';
import renderer from 'react-test-renderer';

 test('Input Component Testing', () => {
    const component = renderer.create(
        <Input
            id={'1'}
            autoComplete={false}
            required={false}
            type={'text'}
            placeholder={'Text Input'}
            className={'Text Input'}
            disabled={'false'}
            maxLength={'50'}
            value={'Text Input'}
            readonly={false}>
        </Input>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});