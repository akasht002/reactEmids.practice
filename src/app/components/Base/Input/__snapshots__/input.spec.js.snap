// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input Component Testing 1`] = `
<div
  className="form-group"
>
  <label
    className="m-0"
  />
  <span>
    <input
      autoComplete={false}
      className="Text Input"
      disabled="false"
      placeholder="Text Input"
      required={false}
      type="text"
      value="Text Input"
    />
  </span>
  <small
    className="form-text text-muted undefined"
    id="undefinedHelp"
  />
</div>
`;
